packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2285,8 +2320,17 @@ mixin WidgetInspectorService {
   /// [WidgetInspectorService] can track which events were dispatched by
   /// overriding this method.
   @protected
-  void postEvent(String eventKind, Map<Object, Object?> eventData) {
-    developer.postEvent(eventKind, eventData);
+  void postEvent(String eventKind, Map<Object, Object?> eventData, {String stream = 'Extension'}) {
```

Comment:
use flutter trailing comma style.

Reply:
The linter seems like it doesn't allow trailing commas here

Reply:
oh I definitely misunderstood which trailing comma we were talking about then 😜